### PR TITLE
docs changes for release 1.7

### PR DESCRIPTION
**Pull request by rsokl**
- adds docs for new `nnet` features
- adjusts input arg names for `softmax_focal_loss` to be consistent with other losses

### CODE REVIEW

File: docs/source/nnet.rst [hunk 1]
```diff
@@ -47,6 +50,22 @@ Activations
...
 
+.. currentmodule:: mygrad.nnet.initializers
+
+Initializers
+------------
+.. autosummary::
+   :toctree: generated/
+
+
+   focal_loss
```
Comment (davidmascharka) [inline docs/source/nnet.rst:61]: That's not an initializer

File: docs/source/nnet.rst [hunk 2]
```diff
@@ -47,6 +50,22 @@ Activations
...
+   glorot_normal
+   glorot_uniform
+   he_normal
+   he_uniform
+   negative_log_likelihood
```
Comment (davidmascharka) [inline docs/source/nnet.rst:66]: Also a loss

File: docs/source/nnet.rst (hunk 1 above)
Comment (rsokl) [reply docs/source/nnet.rst]: Whoops!

File: docs/source/nnet.rst (hunk 2 above)
Comment (rsokl) [reply docs/source/nnet.rst]: Whoops again!

**Pull request merged**